Makefile

```diff
@@ -51,6 +52,21 @@ MD_FILES := $(shell \
 		-name '*.md' \
 	| sed 's/ /\\ /g' \
 )
+UI_FILES = $(shell \
```

Comment:
I understand why do we exclude tests, but why do we exclude the rest? Is it just performance, or is including them in the stats have other side effects?